products/sle/main.pm

```diff
@@ -887,7 +887,9 @@ sub load_x11tests {
     loadtest "x11/xterm";
     loadtest "x11/sshxterm";
     if (gnomestep_is_applicable()) {
-        loadtest "update/updates_packagekit_gpk" unless is_staging;
+        if (!sle_version_at_least('15') && !(is_staging)) {
```

Comment:
I think this is a very bad approach. How do we make sure we will re enable the test later when it does not fail or soft fail here? Better call the test module anyway and return with a soft fail in there

Reply:
PR #3670 created.

Reply:
thank you